flaml/ml.py

```diff
@@ -162,14 +268,6 @@ def sklearn_metric_loss_score(
             score += 1
         else:
             score = 1 - ndcg_score([y_true], [y_predict])
-    else:
-        raise ValueError(
-            metric_name + " is not a built-in metric, "
-            "currently built-in metrics are: "
-            "r2, rmse, mae, mse, accuracy, roc_auc, roc_auc_ovr, roc_auc_ovo,"
-            "log_loss, mape, f1, micro_f1, macro_f1, ap. "
-            "please pass a customized metric function to AutoML.fit(metric=func)"
-        )

```

Comment:
Recover and add "ndcg", "ndcg@k".

Reply:
I have it inside of the check for huggingface. 

Outside there's     
     if is_in_sklearn_metric_name_set(metric_name):
        return sklearn_metric_loss_score(
            metric_name, y_predict, y_true, labels, sample_weight, groups
        )

Which includes ndcg@k. So if the metric is outside of the sklearn metric, it won't enter sklearn_metric_loss_score.